docs/ABOUT.md

```diff
@@ -0,0 +1,5 @@
+Rust is a systems programming language focused on three goals: safety, speed, and concurrency. Rust does not use a garbage collector, but instead ensures safety and optimizes code at compile time. The concept of _ownership_ is how Rust achieves its largest goal, memory safety.
```

Comment:
Oops, merged before reading this.  This is explained in "The Book".  Maybe adding a link to that chapter would be helpful.